static/js/biomedical/landing/biomedical_landing.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Entrypoint for Biomedical DC Landing Page */
+
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { BiomedicalSearchBox } from "./biomedical_search_box";
+import { CardProps, CardTheme } from "./card";
+import { CardWall } from "./card_wall";
+
+// Search url for biomedical searches
+const BIOMEDICAL_SEARCH_URL = "/explore/#q=";
+
+// Mapping of categories to color theme to apply to cards in that category
+const CATEGORIES_TO_THEME = {
+  Diseases: "blue",
+  Ecology: "muted-blue",
+  Pharmacology: "red",
+};
+
+// Order to show card categories in, from left to right
+const CARD_CATEGORY_ORDER = ["Ecology", "Diseases", "Pharmacology"];
```

Comment:
and this could also go into the config where the config just returns a list of category objects like:

```
[
{title: "Ecology",
themeColor: "muted-blue",
questions: ["question1",...]
},
...
]
```

Reply:
Thanks! I like this. Swapped to putting these in the config as a list.